common/kafka/kafka_watcher.cpp

```diff
@@ -24,7 +24,7 @@
 
 #include "boost/functional/hash.hpp"
 #include "common/stats/stats.h"
-#include "common/timeutil/timeutil.h"
+#include "common/timeutil.h"
```

Comment:
any idea why the compiling didn't fail previously? we need to include the code into the build.

Reply:
I didn't realize it's not compiled. Added all the kafka files as a library now.